ui/src/components/list-view/QasListView.vue

```diff
@@ -148,7 +151,7 @@ export default {
     },
 
     async refresh (done) {
-      await this.fetchList()
+      await this.fetchHandler({ ...this.context, url: this.url }, this.fetchList)
```

Comment:
será q não tem outra forma de chamarmos q fique melhor? é copy paste da função/parâmetros 3x

Reply:
talvez seja uma boa, será q não vai ficar muitos métodos "fetch" q no final fazem a mesma coisa? mas ao msm tempo tem mt copy paste, to na dúvida